src/servers/src/http/event.rs

```diff
@@ -123,10 +124,28 @@ pub async fn add_pipeline(
         .insert_pipeline(&pipeline_name, content_type, &payload, query_ctx)
         .await;
 
-    result.map(|_| "ok".to_string()).map_err(|e| {
-        error!(e; "failed to insert pipeline");
-        e
-    })
+    result
+        .map(|pipeline| {
+            let mut headers = HeaderMap::new();
+            headers.append(CONTENT_TYPE, "application/json".parse().unwrap());
+            // Safety check: unwrap is safe here because we have checked the format of the timestamp
+            let version = pipeline
+                .0
+                .as_formatted_string(
+                    "%Y-%m-%d %H:%M:%S%.fZ",
+                    // Safety check: unwrap is safe here because we have checked the format of the timezone
+                    Some(Timezone::from_tz_string("UTC").unwrap()).as_ref(),
```

Comment:
How about use UNIX_MILLS format(like '1718768595191') as version. It's kinda strange to me using a formatted date string(like '2024-06-19 03:43:15.191') a version number.

Reply:
I think it's like the `last-modified-since`.

Reply:
The string is later used as a version parameter to find target pipeline(`pipeline_name` + `version`, `latest` if `version` is `None`), rather then a display-only information.

https://github.com/GreptimeTeam/greptimedb/blob/01e3a24cf77d748e4e4cfdf4155d6e2e2a3aabd6/src/servers/src/http/event.rs#L187-L198

Reply:
I considered the unix timestamp format, but for the user I wanted him to make it simple for the user to understand the content. unix timestamps are pretty standard and simple, and I think both would work. But for user readability I chose the formatted time string.

Reply:
One issue here is that date string is formatted with timezone offset. For example if I were in +0800 and I got result string back in UTC, it would be confusing.

Reply:
Do you mean to use the same formatter for parse and format?

Reply:
The version is `2024-06-19 09:24:55.788204038Z` . 
The "Z" at the end means it's in UTC time. If I use timestamp I need to convert it to even know the exact time.

Reply:
Agree with string format, it's more readable.

Reply:
Ok, let's keep it.